docs/cli.md

```diff
@@ -1082,6 +1082,65 @@ The `whoami` command returns the Viam user for an authenticated CLI session, or
 viam whoami
 ```
 
+### `auth-app`
+
+The `auth-app` command allows you to register and update your mobile or web app (created with the Viam Flutter or TypeScript [SDKs](/sdks/)) with [FusionAuth](https://fusionauth.io/) (the tool Viam uses for authentication and authorization) so that you or other users can log into your app with the same credentials they use to log into the [Viam app](https://app.viam.com).
```

Comment:
Does this mean "you or other users in your org"? Might be nice to specify that if so + update next sentence so that it can be easily understand what input creds (--org-id ?) translate to user permissions 

(if I'm confused, maybe it's just me but could also update for that confusion)

Reply:
No, it doesn't mean that. What this is meant to convey is that whatever perms you have when you log into app.viam.com, you have those exact same perms to orgs, locations and machines when you log into anywhere else (such as a mobile app) through the same FusionAuth login system. My understanding is that the org ID parameter is just to say which org the app belongs to, but someone from a different org could log in.

Reply:
@sguequierre This got buried because I updated it with one of your other suggestions but I didn't mean to resolve it if you have more thoughts!